nncf/experimental/openvino_native/quantization/algorithms/min_max/openvino_backend.py

```diff
@@ -46,8 +57,12 @@
 class OVMinMaxAlgoBackend(MinMaxAlgoBackend):
 
     @property
-    def layers_with_weights_metatypes(self) -> List[OperatorMetatype]:
-        return GENERAL_WEIGHT_LAYER_METATYPES
+    def mat_mul_metatype(self) -> OperatorMetatype:
+        return OVMatMulMetatype
+
+    @property
+    def softmax_metatype(self) -> OperatorMetatype:
```

Comment:
I haven't found any place, where it's used.

Reply:
Removed redundant code